### ISSUE TITLE
GPIO plugin config

### ISSUE THREAD

**Issue by angeloboraine** [OpenLightingProject/ola#1628]
Hi

I'm struggling to get an ola universe to trigger a relay attached to a gpio pin. Whenever I enter what pins I want to control in the config file and refresh the plugin, it  goes red and isn't active anymore. 

When I leave what pins I want to control out it says its active e.g:
<img width="222" alt="Screenshot 2020-04-03 at 02 16 30" src="https://user-images.githubusercontent.com/63079902/78311531-637a9480-7551-11ea-95d2-698cfcd31094.png">
But when I try and enter any pin to control it deactivates e.g:
<img width="195" alt="Screenshot 2020-04-03 at 02 18 02" src="https://user-images.githubusercontent.com/63079902/78311603-97ee5080-7551-11ea-8fa1-78aeb12921db.png">

Please tell me if i'm doing anything wrong. Can you specify what pins by simply writing the numbers in or is there a special way of specifying? And do you need some sort of comma or something to space them out?

Thanks

**Comment by peternewman** [comment]
Can we have some logs please:
https://www.openlighting.org/ola/get-help/ola-faq/#How_do_I_get_olad_-l_4_logs

Have you done the config here:
https://wiki.openlighting.org/index.php/OLA_Device_Specific_Configuration#General_Purpose_I.2FO

Although if it's starting with no pins configured whatsoever that's probably technically a bug!

**Comment by BytePhilHD** [comment]
Hi,

I have the exact same bug. When there is no pin set (like on @angeloboraine first picture), in the ola webinterface on plugins there is gpio active (see picture below). When I'm changing the "gpio_pins" value in the config to any number and reload the plugins the webinterface says "Active No". 


![image](https://user-images.githubusercontent.com/63234545/140329554-564fe448-85da-4e6c-a5f3-c0f5daf41b44.png)

Is there any way to solve the problem?

Thanks

**Comment by peternewman** [comment]
@BytePhilHD I don't know where your comment went (maybe you deleted it)? But like @angeloboraine we need some logs really (as well as details of which version you're running) to try and see what's going on.

Can you also try stopping olad, editing the config files, then starting olad and seeing if that helps?

**Comment by BytePhilHD** [comment]
@peternewman yes I deleted it. The solution is to give olad the permissions for the specific port like on the wiki

https://wiki.openlighting.org/index.php/OLA_Device_Specific_Configuration#General_Purpose_I.2FO

**Comment by peternewman** [comment]
Ah great, glad you sorted it. Did OLAd give any sane error messages in it's logs to make this easier to find in future?

@angeloboraine does this fix it for you too?
